WorkloadManagementSystem/JobWrapper/JobWrapperTemplate.py

```diff
@@ -162,7 +162,7 @@ def execute( arguments ):
 
   if 'OutputSandbox' in arguments['Job'] or 'OutputData' in arguments['Job']:
     try:
-      result = job.processJobOutputs( arguments )
+      result = job.processJobOutputs()
```

Comment:
This change also needs to be reflected in the extensions no ?

Reply:
I do not think that any extension overrides the JobWrapper. Am I wrong ? LHCb ?

Reply:
Oups sorry, I mixed it with the pilot wrapper template !